Player/VideoCloudBasicPlayer/objc/VideoCloudBasicPlayer/ar.lproj/LaunchScreen.strings

```diff
@@ -0,0 +1,6 @@
+
+/* Class = "UILabel"; text = "Copyright © 2020, Brightcove, Inc. All rights reserved."; ObjectID = "8ie-xW-0ye"; */
```

Comment:
Oh, if only you had gone for 2021.  :)  It's no biggie. No need to fix it unless you feel the urge.